### DISCUSSION TITLE
Cron job that is less frequent overnight

### DISCUSSION THREAD

**Discussion by berkserbet** [community/community#106525] [Actions] [answered]
### Select Topic Area

Question

### Body

I would like my job to run every 15 minutes between hours 8-18 then every 60 minutes. How can I do this? Or do I need 2 cron jobs?

**Comment by ghost** [comment]
For that you probably need two separate cron jobs. Here's how you could set them up:

1. **Cron Job for 8:00 to 18:00**: This job will run every 15 minutes during these hours.
   - Cron expression: `*/15 8-18 * * *`
   - This translates to: "At every 15th minute past every hour from 8 through 18."

2. **Cron Job for Outside 8:00 to 18:00**: This job will run every 60 minutes outside the 8:00 to 18:00 window, which means it will run at 18:00, and then every hour until 8:00 the next day.
   - Cron expression for hours after 18:00: `0 19-23 * * *`
   - Cron expression for hours before 8:00: `0 0-7 * * *`
   - These translate to: "At minute 0 past every hour from 19 through 23" and "At minute 0 past every hour from 0 through 7," respectively.

**Reply by berkserbet** [reply]
This makes sense, thanks!

**Comment by DevDaniloFerrari** [comment]
To achieve this scheduling using a single cron job, specify a combination of minute and hour values in the cron expression. Here's an example:


```
*/15 8-18 * * *   # Every 15 minutes between 8:00 AM and 6:00 PM
0    9-17 * * *   # On the hour (every 60 minutes) between 9:00 AM and 5:00 PM
```

By combining these two expressions, you cover the requirement of running every 15 minutes between 8 AM and 6 PM and every 60 minutes during the remaining hours.

**Reply by berkserbet** [reply]
Amazing! How would I add this to my yml file?

**Reply by DevDaniloFerrari** [reply, marked as answer]
To add the specified cron job scheduling to a YAML file, you can use the following structure:

```yaml
jobs:
  - name: my-job
    schedule: "*/15 8-18 * * *"
    command: "/path/to/your/command"
  - name: my-hourly-job
    schedule: "0 9-17 * * *"
    command: "/path/to/your/hourly/command"
